packages/app/src/components/home/HomePage.tsx

```diff
@@ -98,102 +110,105 @@ const HomePage = () => {
-							quality,
-							consistent, and compliant applications for the BC government as well as the supports
-							available to
-							them.</ContainerP>
+				<Grid item sm={12} md={4} style={{display: 'flex'}}>
+					<Container style={{display: 'flex', flexDirection: 'column', justifyContent: 'flex-start'}}>
+					<ContainerImg src="" />
+						<ContainerH2>
+							<Link to="docs/default/component/mobile-developer-guide">
+								Mobile development guide
+							</Link>
+						</ContainerH2>
+						<ContainerP>Detailed guidance on the steps and practices you must follow when developing a mobile application.</ContainerP>
 						<LinkButton
 							className="default-button"
-							title="bcdg"
-							to="docs/default/component/bcdg"
-						>Read Now</LinkButton>
+							title="mobile"
+							to="docs/default/component/mobile-developer-guide"
+						>Review the mobile development guide</LinkButton>
 					</Container>
 				</Grid>
-				<Grid item sm={12} md={4}>
-					<Container>
-						<ContainerImg src="https://digital.gov.bc.ca/wp-content/uploads/2023/03/communityGrey.png"/>
-						<ContainerH2>Find out about deploying applications in BC Government's private
-							cloud.</ContainerH2>
-						<ContainerP>The Private Cloud Technical Documentation describes the services avaialable within
-							BC
-							government OpenShift environments, as well as recommendations on making the most of
-							them.</ContainerP>
+				<Grid item sm={12} md={4} style={{display: 'flex'}}>
+					<Container style={{display: 'flex', flexDirection: 'column', justifyContent: 'flex-start'}}>
+						<ContainerImg src="" />
+						<ContainerH2>
+							<Link to="docs/default/component/platform-developer-docs">
+								Private cloud application deployment
+							</Link>
+						</ContainerH2>
+						<ContainerP>Learn how to deploy applications on the private cloud OpenShift environment.</ContainerP>
 						<LinkButton
 							className="default-button"
 							title="platform-developer-docs"
 							to="docs/default/component/platform-developer-docs"
-						>Read Now</LinkButton>
+						>Get ready to deploy</LinkButton>
 					</Container>
 				</Grid>
 			</Grid>
-			<Grid item xs={12}>
-				<HomePageToolkit title="Community Tools"
-								 tools={tools}
-				/>
-			</Grid>
 
+			<Box sx={{ pt: 6 }}>
+				<Grid item xs={12}>
+					<HomePageToolkit
+						title="Get support from the developer community"
+						tools={tools}
+					/>
+				</Grid>
+			</Box>
+
+			<Box sx={{ pt: 5 }}>
+				<Box sx={{ pb: 1 }}>
+					<Typography variant="h3">
+						Provide feedback
+					</Typography>
+				</Box>
+				<Typography paragraph>
+					The B.C. government DevHub is managed by the Developer Experience team. <a href='mailto:jen.reiher@gov.bc.ca'>Join us</a> as we work together to create impactful solutions by <a href='#'>providing feedback</a> or participating in user research.
```

Comment:
let's make this the developer.experience email please - and attach it to the feedback instead of "join us"

Reply:
Done!